draft-ietf-wish-whip.md

```diff
@@ -266,8 +266,7 @@ WHIP clients generating the HTTP PATCH body with the SDP fragment and its subseq
  - As per {{!RFC9429}}, only m-sections not marked as bundle-only can gather ICE candidates, so given that the "max-bundle" policy is being used, the SDP fragment will contain only the offerer-tagged m-line of the bundle group.
  - The WHIP client MAY exclude ICE candidates from the HTTP PATCH body if they have already been confirmed by the WHIP session with a successful HTTP response to a previous HTTP PATCH request.
 
-If the WHIP session is using entity-tags for identifying the ICE sessions in explained in {{http-patch-usage}}, a WHIP client sending a PATCH request for performing trickle ICE MUST include an "If-Match" header field with the latest known entity-tag as per {{Section 13.1.1 of !RFC9110}}.
-When the PATCH request is received by the WHIP session, it MUST compare the indicated entity-tag value with the current entity-tag of the resource as per {{Section 13.1.1 of !RFC9110}} and return a "412 Precondition Failed" response if they do not match. If the HTTP PATCH request does not contain an "If-Match" header the WHIP session MUST return an "428 Precondition Required" response as per {{Section 3 of !RFC6585}}.
+WHIP sessions that support Trickle ICE MUST return a strong ETag as per {{Section 8.8.3 of !RFC9110}} in a successful response to a POST to the WHIP endpoint. WHIP clients that support Trickle ICE MUST use the ETag as per {{Section 13.1.1 of !RFC9110}} in its PATCH requests. Missing or outdated ETags in the PATCH requests from WHIP clients will be answered by WHIP sessions as per {{Section 13.1.1 of !RFC9110}} and {{Section 3 of !RFC6585}}, with a "428 Precondition Required" response for a missing entity tag, and a "412 Precondition Failed" response for a non-matching entity tag.
```

Comment:
we already have the following text in the `{{http-patch-usage}}`:

```
The WHIP client MAY send overlapping HTTP PATCH requests to one WHIP session. Consequently, as those HTTP PATCH requests may be received out-of-order by the WHIP session, if WHIP session supports ICE restarts, it MUST generate a unique strong entity-tag identifying the ICE session as per {{Section 8.8.3 of !RFC9110}}, being OPTIONAL otherwise. The initial value of the entity-tag identifying the initial ICE session MUST be returned in an ETag header field in the "201 Created" response to the initial POST request to the WHIP endpoint.

WHIP clients SHOULD NOT use entity-tag validation when matching a specific ICE session is not required, such as for example when initiating a DELETE request to terminate a session. WHIP sessions MUST ignore any entity-tag value sent by the WHIP client when ICE session matching is not required, as in the HTTP DELETE request.
```

I would probably move the 

```
Missing or outdated ETags in the PATCH requests from WHIP clients will be answered by WHIP sessions as per {{Section 13.1.1 of !RFC9110}} and {{Section 3 of !RFC6585}}, with a "428 Precondition Required" response for a missing entity tag, and a "412 Precondition Failed" response for a non-matching entity tag.
```
 to the http-patch-usage section and just leave here:

``
WHIP sessions and clients that support Trickle ICE MUST make use of entity-tags and conditional requests as explained in {{http-patch-usage}}.
```

Reply:
Good idea. I think that works well.

I think the only information "lost" by that change is the explicit reference for WHIP clients to section 13.1.1 of RFC9110. But that reference still exists in the new sentences about missing or outdated etags. So I don't think it's a big deal.

Reply:
Sounds good to me!